core/src/tangram.cpp

```diff
@@ -75,9 +82,11 @@ class Map::Impl {
     TileWorker tileWorker{MAX_WORKERS};
     TileManager tileManager{tileWorker};
     MarkerManager markerManager;
+    FrameBuffer selectionBuffer{256, 256};
```

Comment:
This is the size being used in tangram-js. This seems to give enough resolution but I agree that having this a ratio of the actual rendering area would make sense.
@bcamper how did this value was actually chosen in `-js`?